nevergrad/optimization/test_optimizerlib.py

```diff
@@ -898,3 +898,13 @@ def test_pymoo_batched() -> None:
             loss = losses.pop()
             optimizer.tell(x, loss)
     assert len(optimizer._current_batch) == 0  # type: ignore
+
+
+def test_bounded_cma() -> None:
+    x = (
+        ng.optimizers.CMA(ng.p.Array(shape=(7,)).set_bounds(-12, 12), budget=1000)
```

Comment:
This syntax is much better because it allows to automatically set the init (in the middle), and sigma (as a fraction of the span)
```suggestion
        ng.optimizers.CMA(ng.p.Array(shape=(7,) lower=-12, upper=12), budget=1000)
```